src/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
```

Comment:
```suggestion
 * Copyright 2024 Google LLC
```

Unless you wrote this last year I guess?

Reply:
Yep, I started this the week before the holidays.